src/test/java/perosnal/spotifymatcher/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,80 @@
+package perosnal.spotifymatcher.repository;
+
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.dao.InvalidDataAccessApiUsageException;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import perosnal.spotifymatcher.TestToken;
+import perosnal.spotifymatcher.model.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class UserRepositoryTest {
+
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Test
+    void shouldPersistUser() {
```

Comment:
I highly recommend using some form of behavior naming for tests, like:

```suggestion
    void save_givenANewUser_shouldPersistIt() {
```

Reply:
Or 

```suggestion
    void save_whenANewUserIsCreated_willPersistData()
```